app/src/main/res/layout/admin_pin_activity.xml

```diff
@@ -10,105 +10,96 @@
-          android:layout_width="match_parent"
-          android:layout_height="wrap_content"
-          android:layout_marginStart="28dp"
-          android:layout_marginTop="32dp"
-          android:layout_marginEnd="28dp"
-          android:fontFamily="sans-serif"
-          android:text="@string/admin_pin_requirement_description"
-          android:textColor="@color/oppiaPrimaryText"
-          android:textSize="16sp"
-          app:layout_constraintEnd_toEndOf="parent"
-          app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toTopOf="parent" />
+      <View
+        android:layout_width="0dp"
+        android:layout_height="6dp"
+        android:background="@drawable/toolbar_drop_shadow"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
 
-        <TextView
-          android:id="@+id/admin_pin_warning_text"
-          android:layout_width="match_parent"
-          android:layout_height="wrap_content"
-          android:layout_marginStart="28dp"
-          android:layout_marginTop="16dp"
-          android:layout_marginEnd="28dp"
-          android:fontFamily="sans-serif"
-          android:text="@string/admin_pin_pin_description"
-          android:textColor="@color/oppiaPrimaryText"
-          android:textSize="16sp"
-          app:layout_constraintEnd_toEndOf="parent"
-          app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toBottomOf="@+id/admin_pin_main_text" />
+      <TextView
+        android:id="@+id/admin_pin_main_text"
+        style="@style/Subtitle1"
+        android:layout_height="wrap_content"
+        android:layout_width="match_parent"
+        android:layout_marginStart="28dp"
+        android:layout_marginTop="32dp"
+        android:layout_marginEnd="28dp"
+        android:text="@string/admin_pin_requirement_description"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
 
-        <org.oppia.android.app.profile.ProfileInputView
-          android:id="@+id/input_pin"
-          android:layout_width="match_parent"
-          android:layout_height="wrap_content"
-          android:layout_marginStart="28dp"
-          android:layout_marginTop="28dp"
-          android:layout_marginEnd="28dp"
-          app:inputLength="5"
-          app:isPasswordInput="true"
-          app:label="@string/admin_pin_new_pin"
-          app:layout_constraintEnd_toEndOf="parent"
-          app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toBottomOf="@+id/admin_pin_warning_text"
-          profile:error="@{viewModel.pinErrorMsg}" />
+      <TextView
+        android:id="@+id/admin_pin_warning_text"
+        style="@style/Subtitle1"
+        android:layout_height="wrap_content"
+        android:layout_width="match_parent"
```

Comment:
must be in order. 
Try performing a reformat code here from the menu option in the toolbar.

Reply:
Done